src/core.ts

```diff
@@ -73,12 +91,12 @@ export default {
     TransparentNeutral400: 'rgba(203, 211, 223, 0.55)',
     TransparentScrim: 'rgba(0, 0, 0, 0.25)',
     TransparentScrimDarker: 'rgba(0, 0, 0, 0.50)',
-  },
+  } as ColorType,
   Font: {
-    Regular: 'ProximaNovaRegular, Helvetica, Arial, sans-serif',
-    Semibold: 'ProximaNovaSemibold, Helvetica, Arial, sans-serif',
-    Bold: 'ProximaNovaBold, Helvetica, Arial, sans-serif',
-  },
+    Regular: 'Inter, Helvetica, Arial, sans-serif',
```

Comment:
Do we want to match the old naming pattern so this is `InterRegular`, `InterSemibold`, `InterBold`?

Reply:
I was originally planning to get this route, but it looks like `Inter` doesn't use different names like that. It's simply `Inter` with different weights. Unless I'm missing something.

Reply:
So, the name itself is actually dependent on how you import the font into your app. Here is an example of how we are currently importing fonts

```
@font-face {
  font-family: 'ProximaNovaSemibold';
  src: url('https://s3.amazonaws.com/path-to-font/fonts/proxima-nova/semibold.eot');
  font-weight: 600;
  font-style: normal;
}
```

You can see that `font-family` is specified as `ProximaNovaSemibold`, but the font file itself isn't named that.

Reply:
Changed Inter to match the old naming pattern.